tests/roxctl/bats-tests/local/roxctl-netpol-connectivity-map-release.bats

```diff
@@ -430,6 +459,87 @@ payments/gateway[Deployment] => payments/visa-processor[Deployment] : TCP 8080'
   refute_output --partial 'frontend/webapp[Deployment] => backend/shipping[Deployment] : TCP 8080'
 }
 
+@test "roxctl-release netpol connectivity map generates connlist with exposure-analysis for acs-security-demo" {
+  check_acs_security_demos_files
+  run roxctl-release netpol connectivity map "${acs_security_demos_dir}" --exposure
+  assert_success
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  # partial is used to filter connlist and WARN and INFO messages
+  assert_output --partial 'Exposure Analysis Result:
+Egress Exposure:'
+# backend/checkout[Deployment]               =>      entire-cluster : UDP 5353
+# backend/recommendation[Deployment]         =>      entire-cluster : UDP 5353
+# backend/reports[Deployment]                =>      entire-cluster : UDP 5353
+# frontend/webapp[Deployment]                =>      entire-cluster : UDP 5353
+# payments/gateway[Deployment]               =>      entire-cluster : UDP 5353
```

Comment:
hi @adisos ,
in bats tests, i had troubles comparing actual output vs expected output for txt format (the results are correct).
succeeded only with the "header" lines;
the multiple spaces and tabs in the exposure results made it hard to compare 
(i have tried multiple things to solve this, but unfortunately failed to)

- with `md` the output is compared successfully

Reply:
can you consider adding some bash helper function to filter out spaces and then compare?

Reply:
fixed